api/scripts/load_test_data.py

```diff
@@ -0,0 +1,56 @@
+import json
+from pathlib import Path
+
+from sqlalchemy import MetaData, create_engine
+from sqlalchemy.orm import sessionmaker
+
+from mod.src.database import database, db_models
+from mod.src.settings import SETTINGS
+
+file_path = Path.cwd().joinpath("mod").joinpath("fixtures").joinpath("test_data.json")
+
+if not file_path.exists():
+    print("File does not exist")
+    exit(1)
+
+
+def get_class_by_tablename(tablename):
+    base = database.Base
+
+    for mapper in base.registry.mappers:
+        if mapper.class_.__tablename__ == tablename:
+            return mapper.class_
+
+
+class BytesDecoder(json.JSONDecoder):
+    def __init__(self, *args, **kwargs):
+        json.JSONDecoder.__init__(self, object_hook=self.object_hook, *args, **kwargs)
+
+    def object_hook(self, dct):
+        if "hashed_password" in dct:
+            dct["hashed_password"] = bytes.fromhex(dct["hashed_password"])
+            dct["salt"] = bytes.fromhex(dct["salt"])
+            return dct
```

Comment:
This looks a little weirdly hard-coded to me, but honestly I really have no clue what's going on around this